src/client/Microsoft.Identity.Client/TelemetryCore/Internal/Events/ApiEvent.cs

```diff
@@ -30,7 +30,8 @@ public enum ApiIds
             // The API Ids for managed identity will not be found in Http telemetry,
             // as we don't hit eSTS for managed identity calls.
             AcquireTokenForSystemAssignedManagedIdentity = 1015,
-            AcquireTokenForUserAssignedManagedIdentity = 1016
+            AcquireTokenForUserAssignedManagedIdentity = 1016,
```

Comment:
Please add a commetn with the version of when this is introduced. It'll help when querying the data.